actions/release/ReleaseGitHub.ts

```diff
@@ -0,0 +1,182 @@
+          if (line.includes(shortVersion)) {
+            start = i + 2;
+          }
+        } else {
+          // Find the second version heading in changelog, i.e., end
+          end = i;
+          break;
+        }
+      }
+    }
+
+    if (start < 0) {
+      throw new Error(`Version ${shortVersion} not found in changelog`);
+    }
+
+    return [
+      ...this.getHeader(),
+      "**Release Notes**",
+      "------------",
+      ...changelogLines.slice(start, end),
+      ...this.getFooter(version),
+    ].join("\n");
+  }
+
+  async release(version: string, releaseNotes: string): Promise<number> {
+    let existingRelease;
+
+    try {
+      existingRelease = await this.octokit.repos.getReleaseByTag({
+        ...this.context.repo,
+        tag: version,
+      });
+    } catch (error) {
+      core.info(`Could not get existing GitHub release: ${error}`);
+    }
+
+    if (existingRelease) {
+      core.info(`Updating existing GitHub release: ${version}`);
+      const updateReleaseResponse = await this.octokit.repos.updateRelease({
+        ...this.context.repo,
+        release_id: existingRelease.data.id,
+        tag_name: version,
+        name: version,
+        body: releaseNotes,
+      });
+
+      return updateReleaseResponse.data.id;
+    } else {
+      core.info(`Creating GitHub release: ${version}`);
+      const createReleaseResponse = await this.octokit.repos.createRelease({
+        ...this.context.repo,
+        tag_name: version,
+        name: version,
+        body: releaseNotes,
+      });
+
+      return createReleaseResponse.data.id;
+    }
+  }
+
+  async uploadAssets(releaseId: number): Promise<void> {
+    const assetsResponse = await this.octokit.repos.listReleaseAssets({
+      ...this.context.repo,
+      release_id: releaseId,
+    });
+
+    for (const asset of assetsResponse.data) {
+      core.info(
+        `Deleting GitHub release asset: id=${asset.id}, name=${asset.name}`
+      );
+      await this.octokit.repos.deleteReleaseAsset({
+        ...this.context.repo,
+        release_id: releaseId,
+        asset_id: asset.id,
+      });
+    }
+
+    for (const asset of this.params.assets) {
+      core.info(`Reading asset from disk: ${asset}`);
+      const assetContents = readFileSync(asset, "binary");
+      const assetName = path.basename(asset);
+
+      core.info(`Uploading GitHub release asset: ${asset}`);
+      await this.octokit.repos.uploadReleaseAsset({
+        ...this.context.repo,
+        release_id: releaseId,
+        name: assetName,
+        data: assetContents,
+        headers: { "Content-Type": "application/zip" },
+      });
+    }
+  }
+
+  getHeader(): string[] {
+    return [];
+  }
+
+  getFooter(version: string): string[] {
+    let footer: string[] = [];
+
+    if (this.context.repo.owner === "twilio") {
+      footer = [
+        `**[Docs](https://twilio.com/docs/libraries/reference/${this.context.repo.repo}/${version}/index.html)**`,
```

Comment:
don't think this works for go

Reply:
It doesn't, but it's also what's currently happening. Suggestions?

Reply:
Added a case for twilio-go.